layouts/partials/header.html

```diff
@@ -17,12 +17,25 @@
     <link rel="icon" type="image/png" href="{{ "favicon-16x16.png" | relURL }}" sizes="16x16">
     <link rel="icon" type="image/png" href="{{ "favicon-128.png" | relURL }}" sizes="128x128">
     <meta name="application-name" content="Apache Camel">
-    <meta property="og:title" content="Apache Camel helps you integrate over 300 different systems!">
-    <meta property="og:site_name" content="Apache Camel">
-    <meta property="og:url" content="https://camel.apache.org">
-    <meta property="og:description" content="Camel is an open source integration framework that empowers you to quickly and easily integrate various systems consuming or producing data.">
+    
+    <meta property="og:title" content="{{ .Title }}"> 
+    <meta property="og:site_name" content="{{ .Site.Title }}"> 
+    <meta property="og:url" content="{{ .Permalink }}"> 
+    {{ if .Description}}
+        <meta name="og:description" content="{{ .Description }}">
+    {{ else }}
+        {{ if .Params.preview}}
+        <meta name="og:description" content="{{ .Params.preview }}">
+        {{ end }}
```

Comment:
I think we're missing an `{{ else }}` here, if there is no `page.Description` and no `page.Params.preview` defined.